qiskit_optimization/runtime/vqe_program.py

```diff
@@ -309,8 +308,8 @@ class VQEProgramResult(VQEResult):
 
     def __init__(self) -> None:
         super().__init__()
-        self._job_id: Optional[str] = None
-        self._optimizer_history: Optional[Dict[str, Any]] = None
+        self._job_id = None  # type: str
```

Comment:
Why do you need to revert the type hint `Optional[str]` to comment `# type: str`?

Reply:
I am guessing this was a direct copy of the updated module in Nature that looks like that. I know this module was a little different before - presumably since it was added afterwards and had some changes.

Reply:
I see. I prefer type hint personally, but, the code should be consistent with the program in nature. I'm OK with the current code.

Reply:
Yes, @woodsp-ibm is right I copied the code from nature so that we don't maintain two different codes that do the same (until we find a solution where we only have the code once and not copy+pasted in the first place) 🙂